README.md

```diff
@@ -0,0 +1,7 @@
+# camlboot: An OCaml bootstrap experiment
+
+camlboot is an experiment on the boostraping of the OCaml compiler. It is composed of:
+
+- An interpreter of OCaml, in the directory `interpreter/`, which is able to interpret the OCaml compiler. This interpreter is written in a subset of OCaml called miniml, for which a compiler is available as part of the experiment.
+- A compiler for miniml, in the directory `miniml/compiler/`. This compiler compiles miniml to OCaml bytecode, which is then executed by the OCaml runtime. It is written in scheme (more specifically, guile), since the objective is to bootstrap OCaml.
```

Comment:
I think I would add a link to OCaml in the first paragraph, and a link to guile in this bullet point. I would also mention that guile is bootstrapped from gcc, so we're actually removing something from the bootstrap path, not simply shifting the problem of bootstrapping OCaml to bootstrapping Guile :)

Maybe objective -> goal